liver_imaging_analysis/engine/engine.py

```diff
@@ -443,9 +460,11 @@ def test(self, dataloader = None, callback = False):
                     batch[Keys.PRED],
                     batch[Keys.LABEL]
                     ).item()
-                #Apply post processing transforms on prediction
-                batch = self.post_process(batch)
-                self.metrics(batch[Keys.PRED].int(), batch[Keys.LABEL].int())
+                
+                if post_process:
```

Comment:
maybe we should separate mandatory post processing (activation/discretization) from enhancement post processing. It doesn't make sense to skip metric calculation if you don't want to post process the model outputs